rzo.gemspec

```diff
@@ -0,0 +1,48 @@
+# coding: utf-8
+
+lib = File.expand_path('../lib', __FILE__)
+$LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
+require 'rzo/version'
+
+Gem::Specification.new do |spec|
+  spec.name          = 'rzo'
+  spec.version       = Rzo.version
+  spec.licenses      = ['Apache-2.0']
+  spec.authors       = ['Garrett Honeycutt', 'Jeff McCune']
+  spec.email         = ['code@learnpuppet.com']
+
+  spec.summary       = 'Rizzo (rzo) is a tool for working with Vagrant and layered Puppet control repos'
+  spec.description   = 'Rizzo (rzo) is a tool for working with Vagrant and layered Puppet control repos'
+  spec.homepage      = 'https://github.com/ghoneycutt/rizzo'
+
+  # Prevent pushing this gem to RubyGems.org. To allow pushes either set the 'allowed_push_host'
+  # to allow pushing to a single host or delete this section to allow pushing to any host.
+  if spec.respond_to?(:metadata)
```

Comment:
I'm not 100% sure, but it definitely looks that way.  I set the allowed push host to `https://rubygems.org`.